src/_help/publish-documentation/deploy-and-release-management.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Deploy and release management
+---
+
+- TOC
+{:toc}
+
+## What are deployments
+
+Deployment corresponds to the publication of an uploaded version of documentation.
```

Comment:
Should we write "The deployment term" or "A deployment"?
Please remove the `version` term here.
We should also define what a release is, here.